core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildStep.java

```diff
@@ -620,8 +620,15 @@ public NativeImageInvokerInfo build() {
                     nativeImageArgs.add("-H:DebugInfoSourceSearchPath=" + APP_SOURCES);
                 }
                 if (nativeConfig.debugBuildProcess) {
+                    String debugBuildProcessHost;
+                    if (isContainerBuild) {
+                        debugBuildProcessHost = "0.0.0.0";
```

Comment:
How about?
```
String debugAgentAddress = DEBUG_BUILD_PROCESS_PORT;
if (isContainerBuild) {
   debugAgentAddress = "0.0.0.0:" + DEBUG_BUILD_PROCESS_PORT;
}
[...]
.add("-J-Xrunjdwp:transport=dt_socket,address=" + debugAgentAddress + ",server=y,suspend=y");
```

Reply:
We sure can, but what's the benefit of it?
This way we make clear that we only want the process to listen on localhost when building locally, do you forsee any issues with this?

Reply:
No it would just be the more backward compatible way, I guess. Looking at the JDK 11 address parsing code "localhost" is handled fine, so feel free to ignore it.